contracts/message-based/PriceAwareUpgradeable.sol

```diff
@@ -36,7 +36,19 @@ contract PriceAwareUpgradeable is OwnableUpgradeable {
     }
 
 
-    function getPriceFromMsg(bytes32 symbol) internal view returns (uint256) {
+    function getPricesFromMsg(bytes32[] memory symbols) internal view returns (uint32[] memory) {
+        return _getPricesFromMsg(symbols);
+    }
+
+
+    function getPriceFromMsg(bytes32 symbol) internal view returns (uint32) {
+        bytes32[] memory symbols = new bytes32[](1);
+        symbols[0] = symbol;
+        return _getPricesFromMsg(symbols)[0];
+    }
+
+
+    function _getPricesFromMsg(bytes32[] memory symbols) private view returns (uint32[] memory) {
```

Comment:
I know it increases the readibility but each call costs a lot of gas

Reply:
when we call `_getPricesFromMsg` directly, cost is arround 44306. When we use `getPricesFromMsg` cost is 44329